src/app/store.ts

```diff
@@ -1,6 +1,10 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The import statement uses `combineSlices`, which is incorrect. You should import `combineReducers` from `@reduxjs/toolkit` to combine the slices correctly.